python/lsst/ts/ATMCSSimulator/mcs_csc.py

```diff
@@ -244,6 +249,24 @@ def convert_values(name, values, nval):
         max_commanded_position = convert_values(
             "max_commanded_position", max_commanded_position, 5
         )
+        start_position = convert_values("start_position", start_position, 5)
+        for axis in Axis:
+            if max_commanded_position[axis] <= min_commanded_position[axis]:
```

Comment:
Are you sure that the max_commanded_position for any axis cannot be equal to the min_commanded_position for that axis?

Reply:
Good question. I was on the fence about this. I guess I may as well allow equality, though it's a pretty strange axis that has exactly one valid position.